### ISSUE TITLE
Remember me login enabled always by default

### ISSUE THREAD

**Issue by varlesh** [jellyfin/jellyfin-tizen#51]
It's possible disabled on widget? Because not comphortable for multiple accounts and i can't disable this check with remote control BN-1312B
![Screenshot_20200919_075621](https://user-images.githubusercontent.com/8083855/93658648-93a9e700-fa4e-11ea-91da-5fa9a44588a0.png)

**Comment by dmitrylyzo** [comment]
> It's possible disabled on widget? Because not comphortable for multiple accounts

`checked` [here](https://github.com/jellyfin/jellyfin-web/blob/e49b1eaf0dce4df649cf9bab198a0652f11f3449/src/controllers/session/login/index.html#L21), but it is replaced with the value from the settings (after the first login).
If this checkbox is disabled by default, you cannot set autologin from visual form (if there are no passwords). On the other hand, visual login can be intentionally non-automatic. If user has password, you will get manual login form.
So it needs to be discussed.

> i can't disable this check with remote control BN-1312B

A while ago I had to add `Space` handling for Tizen 4: https://github.com/jellyfin/jellyfin-web/blob/e49b1eaf0dce4df649cf9bab198a0652f11f3449/src/elements/emby-checkbox/emby-checkbox.js#L12-L13
Later it turned out that Firefox ignores `preventDefault` and does `click` on checkbox after `keydown`.

Does checkbox turn off and then immediately turn on when you click on it with `OK` button on remote?
Do radiobuttons in sort dialogs (in libraries) work?

**Comment by varlesh** [comment]
Дмитрий, извиняюсь за то, что не владею енглишом. Но не суть))
В общем, на сервере существуют 2 аккаунта - один детский без пароля (мультики, развлекательные передачи), второй взрослый, с паролем (боевики, ужасы и т.д.) Проблема в том, что Jellyfin во-первых, запоминает последнего юзера и автоматически входит при следующем запуске, т.е. если взрослый аккаунт не вышел, то он загрузится в следующий раз. Смысл тогда пароля?
Проблема №2, галочка (Remember Me/Запомить) - у меня получилось ее отключить, но при следующем запуске она опять активна и включена по-умолчанию. Приходится все время выходить перед выключением приложения.

Так вот, суть всего вопроса заключается в том, что если доступны два аккаунта. То почему Jellyfin все время способствует выполнить автовход?! Сохранить сессию, запомнить пароль, не выходить из приложения.... и т.д.
Понимаю, это полезно и удобно для одного аккаунта. Но бессмысленно для двух или более. Ведь зачем тогда нужны другие аккаунты и пароли, если есть автовход?!

**Comment by dmitrylyzo** [comment]
> но при следующем запуске она опять активна и включена по-умолчанию.

Странно, её состояние должно сохраняться в настройках приложения после нажатия на `Вход`.
Переменная `enableAutoLogin` в хранилище.

> Так вот, суть всего вопроса заключается в том, что если доступны два аккаунта. То почему Jellyfin все время способствует выполнить автовход?! Сохранить сессию, запомнить пароль, не выходить из приложения.... и т.д.
Понимаю, это полезно и удобно для одного аккаунта. Но бессмысленно для двух или более. Ведь зачем тогда нужны другие аккаунты и пароли, если есть автовход?!

Например, вход осуществляется на личном устройстве - телефон, планшет. Почему он включен по умолчанию - как говорится, "исторически сложилось" или "кто его знает".
Согласен, с точки зрения безопасности её следует отключить по умолчанию.

Потестил визуальный вход (в браузере): он не сохраняет состояние этого флажка. Но если переменной в хранилище нет, то считается, что автовход включен.
Чтобы поменять логику: [здесь](https://github.com/jellyfin/jellyfin-web/blob/e49b1eaf0dce4df649cf9bab198a0652f11f3449/src/scripts/settings/appSettings.js#L19) заменить `!== 'false'` на `=== 'true'`. Ну и из HTML-ки убрать `checked`, чтобы не моргала.
Остается вопрос: как запомнить вход, используя визуальную форму - для пользователя без пароля. Ещё одна галочка? Или не запоминать совсем?

**Comment by varlesh** [comment]
Да, на ПК все норм, что странно. Галочку если снял, то при следующем запуске она неактивна. Но вот в телевизоре по другому.
Возможно, как вариант, добавить сценарий - если имеется более одного аккаунта, то галочка "запомнить" выключена по дефолту. Ну и соответственно, если один аккаунт, то все запоминается для удобства.
